packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -647,6 +650,46 @@ export function LineItemSublabelAndPrice( { product }: { product: ResponseCartPr
 		);
 	}
 
+	if ( isEnabled( 'difm/allow-extra-pages' ) && isDIFMProduct( product ) ) {
```

Comment:
Do we need to check this flag here? I think we can remove it since what we already block the user in the previous steps and he has no way to select extra steps without that step.

Reply:
You're right, the flag is not required since the PR handles both cases (with and without tiered pricing).